scripts/generate_schemas.js

```diff
@@ -7,7 +7,7 @@ var ISO_DATE_PATTERN = '^(-?(?:[1-9][0-9]*)?[0-9]{4})-(1[0-2]|0[1-9])-(3[01]|0[1
 
 var URL_PATTERN = '(http|https)://[\\w-]+(\\.[\\w-]+)+([\\w.,@?^=%&amp;:/~+#-]*[\\w@?^=%&amp;/~+#-])?';
 
-var JAM_ID_PATTERN = '\\w+\\.\\w+\\.\\w+';
+var JAM_ID_PATTERN = '\\w+';
```

Comment:
FYI, there is some duplication of the session schema in const.js. (I had trouble making node play nice with the ember-specific file, so we stayed with two copies of the schema definition)

Once pending PR ( https://github.com/CenterForOpenScience/experimenter/pull/83 is merged, it might be worth updating the relevant sections to ensure that the new ID patterns are propagated over as necessary.